### PR TITLE
Added dropdown for countries

### PR DESCRIPTION
**Pull request by sourabh1024**
Added dropdown for countries using countries list from ( https://djangosnippets.org/snippets/494/ )

### CODE REVIEW

**Comment by sourabh1024** [comment]
Do I need to do anything for checks failed ? I am new to open source development. Any help will be appreciated. Thanks

File: konfera/models/speaker.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 
+from konfera.models import countries
```
Comment (ZoltanOnody) [inline konfera/models/speaker.py]: `from konfera.models.countries import COUNTRIES` would be nicer

File: konfera/models/speaker.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 from konfera.models.abstract import KonferaModel
 
-
 TITLE_UNSET = 'none'
+COUNTRY_UNSET ='SK'
```
Comment (ZoltanOnody) [inline konfera/models/speaker.py]: `COUNTRY_DEFAULT` would be more descriptive

File: konfera/models/speaker.py [hunk 3]
```diff
@@ -26,7 +29,11 @@ class Speaker(KonferaModel):
     bio = models.TextField(blank=True)
     url = models.URLField(blank=True)
     social_url = models.URLField(blank=True)
-    country = models.CharField(max_length=64)
+    country = models.CharField(
+        choices=countries.COUNTRIES,
+        max_length=64,
```
Comment (ZoltanOnody) [inline konfera/models/speaker.py]: `max_length=2` is enough

**Comment by sourabh1024** [comment]
Review changes added.

File: konfera/models/countries.py [hunk 4]
```diff
@@ -0,0 +1,245 @@
+from django.utils.translation import ugettext_lazy as _
+
```
Comment (ZoltanOnody) [inline konfera/models/countries.py]: Add here one more blank line see https://www.python.org/dev/peps/pep-0008/#blank-lines

File: konfera/models/countries.py [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+    ('YT', _('Mayotte')),
+    ('YU', _('Yugoslavia')),
+    ('ZA', _('South Africa')),
+    ('ZM', _('Zambia')),
+    ('ZR', _('Zaire')),
+    ('ZW', _('Zimbabwe')),
+    ('ZZ', _('Unknown or unspecified country')),
+)
+
+
```
Comment (ZoltanOnody) [inline konfera/models/countries.py]: There are too many blank lines at the end of file

File: konfera/models/speaker.py [hunk 6]
```diff
@@ -1,10 +1,11 @@
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 
+from konfera.models.countries import COUNTRIES
 from konfera.models.abstract import KonferaModel
 
-
```
Comment (ZoltanOnody) [inline konfera/models/speaker.py]: One more blank line, see https://www.python.org/dev/peps/pep-0008/#blank-lines

**Comment by ricco386** [comment]
@sourabh1024 thanks for the PR looks good, just a minor update. As @ZoltanOnody mentioned, please add `country` field to `Location` and `Receipt` jut under `state` and create new migratiom and I will merge it into master. 👍

**Comment by sourabh1024** [comment]
Hey @ricco386 please checkout the last commit , and let me know in case of any concerns.

File: konfera/migrations/0003_auto_20161011_1907.py [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
... (b'SR', 'Suriname'), (b'ST', 'Sao Tome & Principe'), (b'SV', 'El Salvador'), (b'SY', 'Syrian Arab Republic'), (b'SZ', 'Swaziland'), (b'TC', 'Turks & Caicos Islands'), (b'TD', 'Chad'), (b'TF', 'French Southern Territories'), (b'TG', 'Togo'), (b'TH', 'Thailand'), (b'TJ', 'Tajikistan'), (b'TK', 'Tokelau'), (b'TM', 'Turkmenistan'), (b'TN', 'Tunisia'), (b'TO', 'Tonga'), (b'TP', 'East Timor'), (b'TR', 'Turkey'), (b'TT', 'Trinidad & Tobago'), (b'TV', 'Tuvalu'), (b'TW', 'Taiwan, Province of China'), (b'TZ', 'Tanzania, United Republic of'), (b'UA', 'Ukraine'), (b'UG', 'Uganda'), (b'UM', 'United States Minor Outlying Islands'), (b'US', 'United States of America'), (b'UY', 'Uruguay'), (b'UZ', 'Uzbekistan'), (b'VA', 'Vatican City State (Holy See)'), (b'VC', 'St. Vincent & the Grenadines'), (b'VE', 'Venezuela'), (b'VG', 'British Virgin Islands'), (b'VI', 'United States Virgin Islands'), (b'VN', 'Viet Nam'), (b'VU', 'Vanuatu'), (b'WF', 'Wallis & Futuna Islands'), (b'WS', 'Samoa'), (b'YE', 'Yemen'), (b'YT', 'Mayotte'), (b'YU', 'Yugoslavia'), (b'ZA', 'South Africa'), (b'ZM', 'Zambia'), (b'ZR', 'Zaire'), (b'ZW', 'Zimbabwe'), (b'ZZ', 'Unknown or unspecified country')], default=b'SK', max_length=2),
```
Comment (ZoltanOnody) [inline konfera/migrations/0003_auto_20161011_1907.py]: Please make the migrations using `python3` so there is no `b` prefix in `(b'AD', 'Andorra'), ...`

File: konfera/models/location.py [hunk 8]
```diff
@@ -10,6 +13,10 @@ class Location(KonferaModel):
     city = models.CharField(max_length=128)
     postcode = models.CharField(max_length=12)
     state = models.CharField(max_length=128)
+    country = models.CharField(
```
Comment (ZoltanOnody) [inline konfera/models/location.py]: rename the `country` to `state` and remove the `state` above

File: konfera/models/receipt.py [hunk 9]
```diff
@@ -12,6 +15,10 @@ class Receipt(KonferaModel):
     city = models.CharField(max_length=128)
     postcode = models.CharField(max_length=12)
     state = models.CharField(max_length=128)
+    country = models.CharField(
```
Comment (ZoltanOnody) [inline konfera/models/receipt.py]: same here, rename the `country` to `state` and remove the `state` above

File: konfera/admin.py [hunk 10]
```diff
@@ -141,7 +141,7 @@ class LocationAdmin(admin.ModelAdmin):
             'fields': ('title', 'capacity',)
         }),
         (_('Address'), {
-            'fields': ('street', 'street2', 'state', 'city', 'postcode', 'get_here')
+            'fields': ('street', 'street2', 'state', 'city', 'postcode', 'country', 'get_here')
```
Comment (ZoltanOnody) [inline konfera/admin.py]: `country` won't be a field anymore

Comment (sourabh1024) [inline konfera/admin.py]: country or state ? I shall remove state ?

Comment (ZoltanOnody) [inline konfera/admin.py]: Rename the `country` to `state` in models. Then remove `country` here.

Comment (sourabh1024) [inline konfera/admin.py]: Okay, will do so.

**Comment by sourabh1024** [comment]
@ZoltanOnody can you help why the check is failing ?

File: konfera/migrations/0004_auto_20161012_1937.py [hunk 11]
```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('konfera', '0003_auto_20161011_1907'),
+        ('konfera', '0002_schedule_event.py'),
```
Comment (ZoltanOnody) [inline konfera/migrations/0004_auto_20161012_1937.py]: @sourabh1024 the extension `.py` shouldn't be there. It should be just `('konfera', '0002_schedule_event'),`.

Try to change it.

**Comment by ZoltanOnody** [comment]
I think there is a problem with `konfera/migrations/0004_auto_20161012_1937.py` because you first created it and then removed the `0003` migrations. It can be fixed manually but I suggest to remove it completely and then run `python manage.py makemigrations konfera` again. If it won't help, I'll have a look at the test so I know what exactly goes wrong during the tests. :)

**Comment by sourabh1024** [comment]
it worked :)  Thanks @ZoltanOnody

**Pull request merged**